contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -154,55 +153,117 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
+   <ComparisonRow column1={true} column2={false} feature="Custom events" description="Create custom tracking events" />
+   <ComparisonRow column1={true} column2={false} feature="Cohorts" description="Combine users based on properties and events for group analysis" />
+   <ComparisonRow column1={true} column2={false} feature="Formulas" description="Create custom insights using formulas" />
+   <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Release and control features" />
+   <ComparisonRow column1={true} column2={false} feature="A/B testing" description="Run tests on users and groups" />
+   <ComparisonRow column1={true} column2={false} feature="Surveys" description="Gather feedback and book user interviews" />
 </ComparisonTable>
 
-### Pricing
+- Amplitude offers a free Starter plan, but it has many arbitrary limitations and a low limit on usage.
 
-**PostHog** charges for actual usage – i.e. product analytics events, feature flag API requests, and so on. No credit card is required to signup and use PostHog, but entering a credit card unlocks almost all features and **you'll pay nothing** if you stay within our **generous free tier** limits:
+- Users who qualify for [our startup program](/startups) also get $50,000 in PostHog credit and a range of additional benefits. 
 
-|                       | **Free usage per month** 
-|-----------------------|--------------------------
-| **Product analytics** | 1 million events        
-| **Session replay**    | 5,000 recordings         
-| **Feature flags**     | 1 million API requests
-| **A/B testing**       | 1 million API requests  
-| **Surveys**           | 250 responses            
+### Analytics pricing
+
+**PostHog** charges for usage – i.e. analytics events, recorded replays, feature flag API requests, and so on. No credit card is required to signup and use PostHog, but entering a credit card unlocks almost all features and **you'll pay nothing** if you stay within our **generous free tier**
 
 **Amplitude** charges based on monthly tracked users (MTUs). The cost of add owns, such as A/B testing (not available on self-serve plans), group analytics, and unlimited cohorts (self-serve plans are limited to five) isn't disclosed.
 
-Our free tiers are for life, which means you'll always get your first 1 million events (or 5,000 replays and so on) for free every month, no matter how many you end up using.
+<ProductScreenshot
+    imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_vs_amplitude_light_6accf68a1d.jpg" 
+    imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_vs_amplitude_dark_6271c3bfa7.jpg"
+    classes="rounded"
+    alt="product analytics pricing"
+/>
+
+PostHog is consistently cheaper for both product and web analytics. In the above comparison, we're assuming 75 identified events per monthly tracked user (MTU), which is the typical usage for most PostHog users.
```

Comment:
I wonder if it's worth breaking this out for B2C vs B2B users. I _think_ B2C will have a much lower event-to-MTU ratio.

Reply:
I like this idea, but trying to avoid much more complexity atm.